.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2.1
+orbs:
+  node: cypress-io/cypress@1
+jobs:
+  test:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - run: npm ci
+      - run:
+          name: 'Cypress tests'
+          command: npm run test:e2e:ci
+          environment:
+            AIRTABLE_API_KEY: $AIRTABLE_API_KEY
```

Comment:
Not sure if I am missing something but you shouldn't need to define these, if they are setup as environment variables in circle they should just work as is.

Reply:
Assuming its something to do with running cypress? I've seen a lot of weird and wonderful ways of getting it working in circle lol.

Reply:
Yeah we didn't need them defined here good catch!